docs/cluster-configuration.md

```diff
@@ -828,3 +828,22 @@ Labels
     annotation_name="kubevirt.kubevirt.io/jsonpatch"
     severity=info
 ```
+## Log verbosity
+Currently logging verbosity is only supported for Kubevirt.
+
+### Kubevirt
+In order to define logging verbosity for Kubevirt, the following can be done:
```

Comment:
I think this would sound better:
```suggestion
In order to define logging verbosity for Kubevirt, the following can be configured on HyperConverged CR:
```